hooks/useWalletTransaction.tsx

```diff
@@ -38,7 +38,7 @@ const useWalletTransaction = () => {
       return false
     } catch (error) {
       // eslint-disable-next-line no-console
-      console.error(error)
+      console.error(error, "ZIAD ERROR")
```

Comment:
use `handleTxError` for `console.error` on tx handling

Reply:
added ✅